src/asynk.rs

```diff
@@ -402,15 +402,18 @@ impl From<crate::Message> for Message {
 }
 
 impl Message {
-
     /// Creates new empty `Message`, without a Client.
     /// Useful for passing `Message` data, but cannot be used on it's own for associated methods
     /// as those require `Client` injected into `Message`.
-    pub fn new() -> Message {
-        Default::default()
+    pub fn new(subject: &str, data: impl AsRef<[u8]>) -> Message {
```

Comment:
Probably want reply subject as Option too? What about headers?

Reply:
Ok, let's have constructor with all the fields.